core/src/main/java/org/elasticsearch/search/sort/ScriptSortParser.java

```diff
@@ -140,7 +140,6 @@ public SortField parse(XContentParser parser, QueryShardContext context) throws
             sortMode = reverse ? MultiValueMode.MAX : MultiValueMode.MIN;
         }
 
-        // If nested_path is specified, then wrap the `fieldComparatorSource` in a `NestedFieldComparatorSource`
```

Comment:
The commit it was introduced (db421742f7) contained some code that I think it refers to, but that is no longer here. I might be mistaken, but I think its a left-over